qemu/tests/hv_info_check.py

```diff
@@ -0,0 +1,34 @@
+import re
+import logging
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    HyperV Enlightenments info query test:
+
+    1) Boot guest.
+    2) Query supported HyperV Enlightenments by host.
+    """
+    cpu_model_flags = params["cpu_model_flags"]
+
+    vm = env.get_vm(params["main_vm"])
+    error_context.context("Query supported HyperV Enlightenments "
+                          "by host", logging.info)
+    missing = []
+    args = {"type": "full", "model": {"name": "host",
+            "props": {"hv-passthrough": True}}}
+    output = vm.monitor.cmd('query-cpu-model-expansion', args)
+    model = output.get('model')
+    model_prop = model.get('props')
+    cpu_model_flags = cpu_model_flags.replace('_', '-')
+    cpu_model_flags = re.findall(r'hv.*?,', cpu_model_flags)
+    del cpu_model_flags[-9]
```

Comment:
Would you help introduce what's this for?

Reply:
cpu_model_flags[-9] is hv_spinlocks and it is a value  eg: hv_spinlocks=0x1fff, no need to check it.

Reply:
@menli820 The index has chance to be changed, especially when there are new flags introduced, so better approach could be `cpu_model_flags.remove(hv_spinlocks)`

Reply:
@vivianQizhu  changed it as we discussed.